### DISCUSSION TITLE
en/lc/184/

### DISCUSSION THREAD

**Discussion by giscus** [doocs/leetcode#3385] [Announcements]
# en/lc/184/

LeetCode solutions in any programming language

https://leetcode.doocs.org/en/lc/184/

**Comment by luckykumar22** [comment]
SELECT Department.name AS Department ,Employee.name AS Employee, Employee.salary
FROM Department  JOIN Employee  ON Employee.departmentId=Department.id 
WHERE(departmentId, salary) IN
(SELECT departmentId,MAX(salary) FROM Employee GROUP BY departmentId) ;
